### PR TITLE
[Feature]:Changed the hamburger ui with a good animation

### LINKED ISSUE #113: [FEATURE]: Instead of the normal hamburger with a up to down mobile navbar slider create a better looking slider from right to left in the mobile view of every page
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

Yes The problem is by clicking on the hamburger on mobile screen the icons pop up from the top and it looks very bad.
Instead of that we can make a right to left slider with proper spacing between elements and we can close it by clicking on the same hamburger icon.

### Proposed Solution

The solution is that we should change the css of the mobile navbar and set to absolute with right 0 and also will change the zindex of the hamburger so that it can be visible.

### Screenshots

Before:-
Screenshot 2024-03-26 214746

After:-
Screenshot 2024-03-26 214838

### Do you want to work on this issue?

Yes

### If "yes" to above, please explain how you would technically implement this

I have made a website with the same feature.
It would look very awesome if you add the feature

### PR DESCRIPTION
**Pull request by mohit-bhandari45**
Fixes #113

Description:-
Changed the hamburger animation with a proper animation.
Earlier it was sliding from the top which was not looking very good 
But changed it to right slider

How Has This Been Tested?
I have tested it by previewing it multiple times

Screenshots:
Before:-
Screenshot 2024-03-26 214746

After:-
Screenshot 2024-03-26 214838

Types of Changes

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,7 +55,7 @@
             >
           </li>
         </ul>
-        <a href="javascript:void(0);" class="icon" onclick="toggleMobileNav()">
+        <a id="hamburger" href="javascript:void(0);" class="icon" onclick="toggleMobileNav()">
```
Comment (adityaraute) [inline index.html:58]: You've only added this on the index page, but not on the contact or activities page.
Please do so, too. This maintain consistency across all pages.
image

Comment (adityaraute) [inline index.html:58]: Try to make sure that whenever any link in the menu is clicked, the sidebar collapses.
Currently after clicking the `हिन्दी में देखिये` link or `Read in English` link, the sidebar doesn't collapse.

image

**Review by adityaraute — CHANGES_REQUESTED** [review]
Let us know once those changes are done

**Comment by mohit-bhandari45** [comment]
it is done you can review it

**Comment by mohit-bhandari45** [comment]
it is done you can check

**Review by adityaraute — CHANGES_REQUESTED** [review]
You have formatted entire files with `html` and `css` code for some formatting changes.
We cannot merge such a commit. 
These many changes are not acceptable. Please fix the formatting issues and only create a PR for changes that the repo needs.


I suggest you delete this branch, and try making your changes on a new one.
image

**Comment by adityaraute** [comment]
Hi @mohit-bhandari45 
What's the update on this one?

**Pull request closed without merging**